pkg/osbuild/bootc_install_to_filesystem_stage.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 
 	"github.com/osbuild/images/pkg/platform"
+	"golang.org/x/exp/slices"
```

Comment:
We have Go 1.21, `slices` should work just fine.

Reply:
Gonna merge this 'cause I'd like to have it for the partitioning in BIB and open another PR for the import swap.

Reply:
https://github.com/osbuild/images/pull/849